### PR TITLE
machine api cli: code changes

### PR DESCRIPTION
**Pull request by sushmak295**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py [hunk 1]
```diff
@@ -1387,6 +1389,61 @@ def test_aks_nodepool_abort(self, resource_group, resource_group_location):
             self.check('powerState.code', 'Running')
         ])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_machine_cmds(self, resource_group, resource_group_location):
```
Comment (FumingZhang) [inline src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py:1394]: Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=79943087&view=results) to validate the change.

Comment (FumingZhang) [reply src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py:1394]: test passed

**Review by FumingZhang — COMMENTED** [review]
lgtm

Please style check issues and resolve conflicts on history/version if necessary.

https://dev.azure.com/azclitools/public/_build/results?buildId=91678&view=logs&j=ce14e7bd-b9ee-5bd7-1dc4-cb4d70edaf0d&t=90fe1415-141e-56cb-5eb9-d53a9bbb8155&l=3445

File: src/aks-preview/azext_aks_preview/_help.py [hunk 2]
```diff
@@ -1807,6 +1807,44 @@
...
+         type: string
+         short-summary: Name of the agentpool of a managed cluster
+   exmaples:
+       - name: Get information about IP Addresses, Hostname for all machines in an agentpool
+         text: az aks machine list --cluster-name <clusterName> --nodepool-name <apName>
+"""
+
+helps['aks machine show'] = """
+   type: command
+   short-summary: show IP Addresses, Hostname for a specific machine in an agentpool for a managedcluster.
```
Comment (yanzhudd) [inline src/aks-preview/azext_aks_preview/_help.py:1832]:
```suggestion
   short-summary: Show IP Addresses, Hostname for a specific machine in an agentpool for a managedcluster.
```

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**